packages/yourloops/components/chat/chat-message.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2022, Diabeloop
+ * Generic Chat window
```

Comment:
I suspect that this is incorrect as this is the same one that is found in the file `packages/yourloops/components/chat/chat-widget.tsx`